api/CloseEvent.json

```diff
@@ -6,7 +6,7 @@
         "spec_url": "https://websockets.spec.whatwg.org/#the-closeevent-interface",
         "support": {
           "chrome": {
-            "version_added": "13"
+            "version_added": "15"
```

Comment:
"close_event" on `WebSocket` is older, so no change needed there. (If correct, it just means it was a plain `Event` before it was a `CloseEvent`.)